coil-base/src/main/java/coil/fetch/ContentUriFetcher.kt

```diff
@@ -22,10 +26,25 @@ internal class ContentUriFetcher(private val context: Context) : Fetcher<Uri> {
         size: Size,
         options: Options
     ): FetchResult {
+        val inputStream = if (isContactPhotoUri(data)) {
```

Comment:
Opting to keep this logic as part of the `ContentUriFetcher`. I'd like to avoid adding a new `Fetcher` class for this special case.